easybuild/easyconfigs/l/libemf/libemf-1.0.13-GCC-11.3.0.eb

```
@@ -0,0 +1,31 @@
+# J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'ConfigureMake'
+
+name = 'libemf'
+version = '1.0.13'
+
+homepage = 'https://sourceforge.net/projects/libemf/'
+description = "Library implementation of ECMA-234 API for the generation of enhanced metafiles."
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
```

Since `libemf` should be used a dependency for https://github.com/easybuilders/easybuild-easyconfigs/pull/15950 I think it should also be in the `GCCcore` toolchain
```suggestion
toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
```

---

Thanks for finding that, this works here. I have done the suggested changes.